File: an2_sem4/MPP/Homework/sapt78/cs/FestivalSellpoint/FestivalSellpoint.Network/ObjectProtocol/ReserveBiletRequest.cs
using FestivalSellpoint.Network.DTO;
using System;

namespace FestivalSellpoint.Network.ObjectProtocol
{
    [Serializable]
    public class ReserveBiletRequest : IRequest
    {
        public BiletDTO Bilet { get; }
        public ReserveBiletRequest(BiletDTO bilet) => Bilet = bilet;
    }
}

File: an2_sem4/MPP/Homework/sapt11/java/network/src/main/java/ro/ubbcluj/cs/stefnmubb/festivalsellpoint/network/objectprotocol/ReserveBiletRequest.java
package ro.ubbcluj.cs.stefnmubb.festivalsellpoint.network.objectprotocol;

import ro.ubbcluj.cs.stefnmubb.festivalsellpoint.network.dto.BiletDTO;
import ro.ubbcluj.cs.stefnmubb.festivalsellpoint.network.dto.SpectacolDTO;

public class ReserveBiletRequest implements IRequest {
    BiletDTO bilet;

    public ReserveBiletRequest(){}


    public BiletDTO getBilet() {
        return this.bilet;
    }

    public ReserveBiletRequest(BiletDTO bilet) {
        this.bilet = bilet;
    }
}
